### DISCUSSION TITLE
Swift 6 support

### DISCUSSION THREAD

**Discussion by tomwyr** [pointfreeco/swift-clocks#46] [Q&A]
### Description

The package doesn't seem to be compatible with the Swift 6 release. Adding it to my project currently outputs:
```
/Package@swift-6.0.swift:30:23: error: extra argument 'swiftLanguageModes' in call
28 |     ),
29 |   ],
30 |   swiftLanguageModes: [.v6]
   |                       `- error: extra argument 'swiftLanguageModes' in call
31 | )
32 | 

/Package@swift-6.0.swift:30:25: error: reference to member 'v6' cannot be resolved without a contextual type
28 |     ),
29 |   ],
30 |   swiftLanguageModes: [.v6]
   |                         `- error: reference to member 'v6' cannot be resolved without a contextual type
31 | )
32 | 

/Package@swift-6.0.swift:36:6: error: cannot infer contextual base in reference to member 'package'
34 |   // Add the documentation compiler plugin if possible
35 |   package.dependencies.append(
36 |     .package(url: "https://github.com/apple/swift-docc-plugin", from: "1.0.0")
   |      `- error: cannot infer contextual base in reference to member 'package'
37 |   )
38 | #endif in https://github.com/pointfreeco/swift-concurrency-extras
```

### Checklist

- [ ] If possible, I've reproduced the issue using the `main` branch of this package.
- [X] This issue hasn't been addressed in an [existing GitHub issue](https://github.com/pointfreeco/swift-clocks/issues) or [discussion](https://github.com/pointfreeco/swift-clocks/discussions).

**Comment by tomwyr** [comment]
Not a bug but the only available template wouldn't let me select or edit the labels.

**Comment by stephencelis** [comment]
@tomwyr Are you using an old Xcode 16 beta? `swiftLanguageModes` is an official parameter of the Swift 6 release. I'm going to convert this to a discussion since I don't think this is a bug with the library. If you want us to help troubleshoot more we'll need a lot more information to reproduce, including what Xcode or Swift version you're using and maybe a project that reproduces the issue.

**Comment by tomwyr** [comment]
Turns out I had both the beta and the stable versions of Xcode installed. After removing beta, resolving packages and the compilation completed with no issues.
Impressive job on pinpointing the problem. Thanks @stephencelis!
